package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cordova-plugin-image-resizer",
+  "version": "1.0.0",
+  "description": "Cordova / Phonegap plugin for resizing images",
+  "cordova": {
+    "id": "cordova-plugin-image-resizer",
+    "platforms": [
+      "ios",
+      "android"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wymsee/cordova-imageResizer.git"
```

Comment:
Hi man,
Sorry for a late answer, but don't you find it a bit... rude to change the repository URL on the original repository?